### DISCUSSION TITLE
vllm 优化之 cuda_graph 详解 - Zhang

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#200] [General]
# vllm 优化之 cuda_graph 详解 - Zhang

从事 LLM 推理部署、视觉算法开发、模型压缩部署以及算法SDK开发工作,终身学习践行者。LLM_Infercuda graph 解决了可能存在的所有 CPU 开销的来源：如用户编写的逻辑、PyTorch 调度逻辑、内存分配开销以及 GPU 驱动/内核开销（静态图优势）。

https://www.armcvai.cn/2024-11-09/vllm-cuda-graph.html

**Comment by harleyszhang** [comment]
不错，加油！

**Comment by luolun** [comment]
链接失效了，我找到了最新的链接 https://www.armcvai.cn/2024-12-05/vllm-cuda-graph.html
